docs/src/capi/explanation/in-place-upgrades.md

```diff
@@ -0,0 +1,137 @@
+# In-Place Upgrades
+
+Regularly upgrading the Kubernetes version of the machines in a cluster 
+is important. While rolling upgrades are a popular strategy, certain 
+situations will require in-place upgrades:
+
+- Resource constraints (i.e. cost of additional machines).
+- Expensive manual setup process for nodes.
+
+## Annotations
+
+CAPI machines are considered immutable. Consequently, machines are replaced 
+instead of reconfigured.
+While CAPI doesn't support in-place upgrades, {{product}} CAPI does 
+by leveraging annotations for the implementation.
+For a deeper understanding of the CAPI design decisions, consider reading about 
+[machine immutability in CAPI][1], and Kubernetes objects: [`labels`][2], 
+[`spec` and `status`][3].
+
+## Controllers
+
+In {{product}} CAPI, there are two main types of controllers that handle the 
+process of performing in-place upgrades:
+
+- Single Machine In-Place Upgrade Controller
+- Orchestrated In-Place Upgrade Controller
+
+The core component of performing an in-place upgrade is the “Single Machine 
+Upgrader”. It watches for certain annotations on machines and reconciles 
+them to make sure the upgrades happen as expected. As its name suggests, 
+it reconciles each machine individually, considers the machines separately 
+and does not assume any relation between them.
```

Comment:
```suggestion
Upgrader”. The controller watches for annotations on machines and reconciles 
them to ensure the upgrades happen smoothly. 
```
Since the name suggests this we don't need to write it out- I think it becomes clear this is the single machine controller vs in the Orchestrator :)